kube/resources/airbyte-minio.yaml

```diff
@@ -34,12 +34,16 @@ spec:
               value: "minio"
             - name: MINIO_SECRET_KEY
               value: "minio123"
+            - name: MINIO_DOMAIN
+              value: "airbyte-minio-svc"
           ports:
             - containerPort: 9000
           # Mount the volume into the pod
           volumeMounts:
             - name: storage # must match the volume name, above
               mountPath: "/storage"
+      hostname: airbyte-dev-logs
```

Comment:
I think the other error I saw is tied to the config: `STATE_STORAGE_MINIO_BUCKET_NAME`. In our configs, we have two buckets, one for logs, one for states which should to be state of the orchestrator.
I did a quick search, it seems that both are set with the same value `airbyte-dev-logs`. As long as we do not have different values for minio, this should work.

Reply:
Agreed that this is a risk/something that probably needs to be refactored if/when they diverge.